model/src/main/resources/db-update/20060-from-2.5.0-to-2.6.0.sql

```diff
@@ -67,4 +67,64 @@ BEGIN;
     ALTER TABLE buildrecord ADD COLUMN scmbuildconfigrevision varchar(255);
     ALTER TABLE buildrecord ADD COLUMN scmbuildconfigrevisioninternal boolean;
 
+--------------------------------------------------------------------------------
+-- DeliverableAnalyzerReport
+--------------------------------------------------------------------------------
+CREATE TABLE delan_report
+    (
+        operation_id        BIGINT                  NOT NULL,
+        label               VARCHAR(255),
```

Comment:
isn't it called `labels`? And isn't varchar 255 too short?

Reply:
`label` was typo, renamed to `labels`
I've increased the size to 1023. However, I think also 255 was pretty generous, since the largest possible label at this API is "SCRATCH,DELETED", making it the size just 15! (nope, it's not the factorial Honza...)
See 496c0b5

Reply:
Changed from this VARCHAR(1023) to TEXT, see [7071bc3](https://github.com/project-ncl/pnc/pull/3795/commits/7071bc345b99e631fee8c3b089502e702a7c4b27)